### PR TITLE
Remove lima DNS fork

### PR DESCRIPTION
**Pull request by Nino-K**
Related to https://github.com/rancher-sandbox/rancher-desktop/issues/2530 and https://github.com/rancher-sandbox/rancher-desktop-host-resolver/issues/5 

Signed-off-by: Nino Kodabande <nkodabande@suse.com>

### CODE REVIEW

File: test/e2e/dnslookup_test.go [hunk 1]
```diff
@@ -32,13 +32,14 @@ import (
 	"testing"
 	"time"
 
+	"golang.org/x/sys/windows"
```
Comment (mook-as) [inline test/e2e/dnslookup_test.go:37]: The imports here look wong — only the `github.com/rancher-sandbox/` imports should be in a separate section.

**Review by mook-as — CHANGES_REQUESTED** [review]
Lint errors (I haven't actually tried this yet)

**Comment by Nino-K** [comment]
Depends on https://github.com/lima-vm/lima/pull/958

**Review by jandubois — COMMENTED** [review]
There are at least 2 unrelated changes mixed into the commit:

* adding `cmd.SilenceUsage = true` settings
* appending a final `.` to static hostname definitions

They should really be in a separate commit, even if they are piggy-backing on this PR (which I think is fine, given how trivial the changes are).

Otherwise it looks fine to me, but I haven't actually tested it. Leaving final verdict to @mook-as

File: cmd/peer_linux.go [hunk 2]
```diff
@@ -16,11 +16,12 @@ package cmd
 import (
 	"context"
 
+	"golang.org/x/sync/errgroup"
+
 	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/vmsock"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"golang.org/x/sync/errgroup"
```
Comment (mook-as) [inline cmd/peer_linux.go]: The grouping here is odd.
```suggestion
	"github.com/sirupsen/logrus"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"
	"golang.org/x/sync/errgroup"

	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/vmsock"
```

File: cmd/peer_linux.go [hunk 3]
```diff
@@ -43,11 +44,13 @@ AF_VSOCK connections from inside of the WSL VM. It is also a stub DNS forwarder
 | vsock-host | <----- AF_VSOCK -----> [ VM ] <----- AF_VSOCK -----> | vsock-peer   |
  ----------------------------------------------------------------------------------`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
 			// We will always listen for handshake connections from the host (server) in case of restarts
 			go vmsock.PeerHandshake()
 
 			addr := peerViper.GetString("listen-address")
 			tcpPort := peerViper.GetInt("tcp-port")
+
```
Comment (mook-as) [inline cmd/peer_linux.go]: Not sure why we want the extra new line?

**Review by mook-as — CHANGES_REQUESTED** [review]
Code generally looks fine; I'm having issues with E2E tests, though. Will dig more.

**Comment by Nino-K** [comment]
> * appending a final `.` to static hostname definitions

I removed the cobra changes out of this PR, however, adding `.` is a fix for failing tests.

**Review by mook-as — APPROVED** [review]
The changes look fine, but E2E tests hate me :(
Approving anyway, since it seems to work for Nino.

**Pull request merged**